tests/foreman/api/test_new_repository.py

```diff
@@ -0,0 +1,112 @@
+"""Unit tests for the new ``repositories`` paths.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:Assignee: chiggins
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from requests.exceptions import HTTPError
+
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.config import settings
+from robottelo.constants import PRDS
+from robottelo.constants import REPOS
+from robottelo.constants import REPOSET
+
+
+@pytest.mark.skip_if_open('BZ:2137950')
+@pytest.mark.tier1
+def test_negative_disable_repository_with_cv(module_entitlement_manifest_org, target_sat):
+    """Attempt to disable a Repository that is published in a Content View
+
+    :id: e521a7a4-2502-4fe2-b297-a13fc99e679b
+
+    :Steps:
+        1. Enable and sync a RH Repo
+        2. Create a Content View with Repository attached
+        3. Publish Content View
+        4. Attempt to disable the Repository
+
+    :expectedresults: A message should be thrown saying you cannot disable the Repo
+
+    :CaseImportance: Critical
+
+    :CaseAutomation: Automated
+    """
+    rh_repo_id = enable_rhrepo_and_fetchid(
+        basearch='x86_64',
+        org_id=module_entitlement_manifest_org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhst7']['name'],
+        reposet=REPOSET['rhst7'],
```

Comment:
Can we use some EL8 repo here as an example of RH repo, as 6.12+ is EL8 only, non-blocking :)